### ISSUE TITLE
Bot works for 5 minutes then keeps updating stops. 

### ISSUE THREAD

**Issue by CagasAgua** [finndev/PokeBuddy#1166]
I have not found much helpful info on this subject. I am using VPN only one bot, walk speed 9km, not teleporting, staying in my home town, and my account is definitely not soft banned. Any reason or updates as to why this is happening or how to fix it?

**Comment by majdak79** [comment]
hallo cagasAqua, will see you in some days then again with a banned account. I will feel good and sorry about you. good luck

**Comment by Grundy4432** [comment]
I used to have your problem. Here is a solution that you can try. Settings wise. I am walking 10 hm/h, with no incubators on at all.  When I am walking around on this app. I tend to only use one task at a time. Maybe I'll farm pokestops for 3 minutes. Then I will turn that task off, leaving none at at this moment.  Once I see that the task is done in the CURRENT TASK, section which is yellow. I will then move on to another task. Maybe evolve a few things, or transfer pokemon. DO NOT DO ALL THE TASKS AT THE SAME TIME. I did that and that is how I could only originally last 5 min until I got soft banned. Now I'm going for 50 min+. Let me know if this helps.

**Comment by Grundy4432** [comment]
Sorry, didn't read the fact that you said that this wasn't a soft banned. Sorry to hear that. Well if you do another account, try my set up. Maybe it'll help next time.

**Comment by majdak79** [comment]
@Grundy4432 why are you encouriging people for cheating? you know there is no way of cheating in the moment, they will not know? thats not help what you are offering. its like helping to kill someone to help someone ...

**Comment by CagasAgua** [comment]
@majdak79 STFU! what are you doing on a bot forum? You are obviously a cheater yourself

**Comment by Grundy4432** [comment]
@CagasAgua no need to get mad man ^-^, just try what I said, whenever you try to bot an account again haha. I'm sorry that I can't be much help in your current situation.

**Comment by DatHacksGaming** [comment]
@majdak79 That's not true at all... I've cheated on  accounts up to 30+ and none of them are banned yet... one for each team. This also isnt a cheat forum... GitHub is primarily a programmers hang out, you tourists just happen to waltz in from time to time for the new fads lol.
